### DISCUSSION TITLE
List servers with package upgrades pending

### DISCUSSION THREAD

**Discussion by SwimGeek** [saltstack/salt#67505] [Feature Requests]
**Is your feature request related to a problem? Please describe.**

I often use this command to see which servers need package upgrades:
salt '*' pkg.list_upgrades

But, this creates a very long list of all the packages on all the servers.

**Describe the solution you'd like**

It would be nice to run something like:
salt '*' pkg.list_servers_with_upgrades_pending
To only get a list of servers which are in need of upgrades.

**Additional context**

Every time there is a Debian point release, I have to log into each server and upgrade it manually.

**Comment by welcome** [comment]
Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as possible. In the meantime, here’s some information that may help as you continue your Salt journey.
Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community resources including:

  - [Community Wiki](https://github.com/saltstack/community/wiki)
  - [Salt’s Contributor Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html)
  - [Join our Community Slack](https://via.vmw.com/salt-slack)
  - [IRC on LiberaChat](https://web.libera.chat/#salt)
  - [Salt Project YouTube channel](https://www.youtube.com/channel/UCpveTIucFx9ljGelW63-BWg)
  - [Salt Project Twitch channel](https://www.twitch.tv/saltprojectoss)

There are lots of ways to get involved in our community. Every month, there are around a dozen opportunities to meet with other contributors and the Salt Core team and collaborate in real time. The best way to keep track is by subscribing to the Salt Community Events Calendar.
If you have additional questions, email us at saltproject@vmware.com. We’re glad you’ve joined our community and look forward to doing awesome things with you!

**Comment by network-shark** [comment]
Are you talking about something like a summary  ? I think you can do it yourself using a runner.

**Comment by SwimGeek** [comment]
Hi, not sure what a runner is. Do you have an example command?

I'd like a command I can run to list all servers which have one or more upgrades pending.
